notebook.ipynb

```diff
@@ -1,5 +1,12 @@
 {
```

Comment:
<p>According to the Environmental DS Book guidelines, acronyms should be spelled out on first mention. It would probably be worth including the table of abbreviations from the paper, somewhere within this Notebook.</p><p> </p><p>Please update authors with affiliation and GitHub aliases</p><p> </p><p>Consider adding references for models used e.g. CMIP6 (Eyring et al., 2016), or alternatively aknowledging the contributions of the authors referenced within Bône et al. (2022)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eds-book-gallery/CI2023-RC-team2/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Implemented in <a href="https://github.com/eds-book-gallery/CI2023-RC-team2/blob/Notebook-Final/notebook.ipynb" target="_blank">https://github.com/eds-book-gallery/CI2023-RC-team2/blob/Notebook-Final/notebook.ipynb</a></p>

Reply:
closing conversation, resolved